salt/data-pipeline/ejp-to-json-converter.sls

```diff
@@ -40,14 +40,14 @@ temp dir symlink:
         - target: /ext/tmp/data-pipeline-ejp-to-json-converter-.temp
         #- makedirs: True # creates dirs to symlink, not target
         - require:
-            - install repo
+            - ejp-to-json-converter repo
 
-install support repo:
+ejp-csv-deposit flow support repo:
     builder.git_latest:
         - name: git@github.com:elifesciences/data-pipeline-ejp-csv-deposit
         - identity: {{ pillar.elife.projects_builder.key or '' }}
-        - rev: {{ salt['elife.rev']() }}
-        - branch: {{ salt['elife.branch']() }}
+        - rev: master
```

Comment:
This should no longer be configurable?

Reply:
I didn't see we were using this repository too in this PR, I thought we were going to integrate it later and/or it was directly checked out by Nifi

Reply:
I'd start with `master`; it's probably what it was doing before due to the lack of an explicit `revision` for this stack, that lacks a "main" project repository

Reply:
We do use the repo (not as a Docker image). NiFi will call scripts in it (but I don't think it will check code out).
Okay, fair enough. We don't have a full staging environment at the moment. So we could look into that then.